AlphaWallet/Tokens/ViewControllers/TokensViewController.swift

```diff
@@ -368,21 +375,20 @@ class TokensViewController: UIViewController {
     }
 
     private func handleTokenCollectionUpdates() {
-        tokenCollection.subscribe { [weak self] result in
-            guard let strongSelf = self else { return }
-            switch result {
-            case .success(let viewModel):
+
+        tokenCollection.tokensViewModel
+            //.subscribe(on: DispatchQueue.main)
```

Comment:
Should this be uncommented?

Reply:
in case if we use `subscribe(on: )` here we are facing an empty tokens view right before update, as table view updates async, removing updates table view immediatelly

Reply:
Not sure I understand. But would you added a comment to that piece of code? Intutitively, since it's going to update the UI, the reader will probably wonder why there's no `.subscribe(on: DispatchQueue.main)`.